src/legacy/core_plugins/kibana/public/visualize/wizard/_dialog.scss

```diff
@@ -5,6 +5,11 @@
   background-size: 37%;
 }
 
+.visNewVisSearchDialog {
+  min-height: 480px;
+  max-width: 810px;
```

Comment:
The width of this one doesn't need to match the chart type selection and since modal's already have a max-width, this can be removed.